publish_binary_on_windows.bat

```diff
@@ -3,11 +3,11 @@ setlocal EnableDelayedExpansion
 git rev-parse --is-inside-work-tree
 if errorlevel 1 exit /b %errorlevel% 
 
-call npm --no-git-tag-version version patch
+call npm install
 if errorlevel 1 exit /b %errorlevel%
 
-call npm install @sb/prades
```

Comment:
I agree this should not be there.  Having nodes, npm, and prades installed globally should be a pre requisite - configuration